substrate/frame/society/src/lib.rs

```diff
@@ -431,7 +435,7 @@ pub struct MemberRecord {
 }
 
 /// Information concerning a member.
-#[derive(Encode, Decode, Clone, PartialEq, Eq, RuntimeDebug, TypeInfo, Default)]
+#[derive(Encode, Decode, Clone, PartialEq, Eq, RuntimeDebug, TypeInfo, Default, MaxEncodedLen)]
```

Comment:
This should work because `PayoutRecord<Balance, PayoutsVec>` is only used in `PayoutRecordFor<T, I>` where `PayoutsVec` is a `BoundedVec`